STEP_Project/content.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Function that will later have Add/Remove functionality
+ * @param {JS event} event - expect click event
+ */
+function arButtonClick (event) 
+{
+    if (!event.target.matches('BUTTON')) return
+    alert("hello");
+}
+
+/**
+ * This adds a button to every row in the table
+ * If the row already has a button, it skips that row
+ * @param {HTML Object} OUs - List of OUs
+ * @param {integer} numRows - Number of rows
+ */
+function addButtons(OUs, numRows)
+{
+    for (let i = 1; i < numRows; i++) 
```

Comment:
Add a comment as to why we are skipping iterating through numRows - 1 and not numRows.

Reply:
Comment added.